.github/workflows/testing.yml

```diff
@@ -111,6 +111,7 @@ jobs:
         export PYPI_TOKENS=$(mktemp)
         cat > ${PYPI_TOKENS} <<EOF
         examples/shouldi=${{ secrets.PYPI_SHOULDI }}
+        model/tensorflow=${{ secrets.PYPI_MODEL_DAAL4PY }}
```

Comment:
```suggestion
        model/daal4py=${{ secrets.PYPI_MODEL_DAAL4PY }}
```

Reply:
Ah, man. Sorry about this. :disappointed: 
You really have a good eye to catch this kind of bugs so fast tho. :D